core-db/src/main/java/io/novafoundation/nova/core_db/migrations/58_59_ChainNetworkManagement.kt

```diff
@@ -0,0 +1,12 @@
+package io.novafoundation.nova.core_db.migrations
+
+import androidx.room.migration.Migration
+import androidx.sqlite.db.SupportSQLiteDatabase
+import io.novafoundation.nova.core_db.model.chain.ChainLocal
+
+val ChainNetworkManagement_58_59 = object : Migration(58, 59) {
+
+    override fun migrate(database: SupportSQLiteDatabase) {
+        database.execSQL("ALTER TABLE chains ADD COLUMN `isCustomNetwork` INTEGER NOT NULL DEFAULT ${ChainLocal.DEFAULT_NETWORK_BY_DEFAULT_STR}")
```

Comment:
I suggest to make it an enum, similarilly to what we have in the Chain.Asset: `enum Source { DEFAULT, CUSTOM }`. This might prevent us to not to write migrations in future if we will add new types of sources